core/modules/book/book.module

```diff
@@ -405,6 +393,26 @@ function book_form_node_form_alter(&$form, &$form_state, $form_id) {
         'css' => array(backdrop_get_path('module', 'book') . '/css/book.admin.css'),
       ),
     );
+    if (!empty($node->nid)) {
+      $form['book']['actions'] = array(
+        '#theme' => 'links',
+        '#links' => array(),
+        '#attributes' => array('class' => array('action-links')),
+        '#weight' => 40,
+      );
+      if (_book_node_is_removable($node)) {
+        $form['book']['actions']['#links']['remove'] = array(
+          'title' => t('Remove from book outline'),
+          'href' => 'node/' . $node->nid . '/outline-remove',
+        );
+      }
+      if (!empty($node->book['bid'])) {
```

Comment:
This also needs a permission check, if the current user actually can access the reorder page.

Reply:
Done